tests/conftest.py

```diff
@@ -126,32 +126,6 @@ def test_with_any_microvm(test_microvm_any):
 METRICS = get_metrics_logger()
 
 
-def pytest_configure(config):
-    """Pytest hook - initialization"""
-    config.addinivalue_line("markers", "nonci: mark test as nonci.")
-
-
-def pytest_addoption(parser):
-    """Pytest hook. Add command line options."""
-    parser.addoption("--nonci", action="store_true", help="run tests marked with nonci")
-
-
-def pytest_collection_modifyitems(config, items):
-    """Pytest hook. Skip some tests."""
-    skip_markers = {}
-
-    for skip_marker_name in ["nonci"]:
-        if not config.getoption(f"--{skip_marker_name}"):
-            skip_markers[skip_marker_name] = pytest.mark.skip(
-                reason=f"Skipping {skip_marker_name} test"
-            )
-
-    for item in items:
-        for skip_marker_name, skip_marker in skip_markers.items():
-            if skip_marker_name in item.keywords:
-                item.add_marker(skip_marker)
```

Comment:
Nice simplification

Reply:
sadly it doesn't work :sob: there's some other fixtures that explicitly check for the --nonci marker where I couldn't find a solution - so its back to this for now